common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.sh

```diff
@@ -21,13 +21,17 @@
 #   VENV_DIR=jax-models.venv
 #   PYTHON=/usr/bin/python3.10
 #   WITH_CUDA=1
+#
+# Positional arguments:
+#   FILTER (Optional): Regex to match models, e.g., BERT_LARGE_FP32_.+
 
 set -xeuo pipefail
 
 TD="$(cd $(dirname $0) && pwd)"
 VENV_DIR="${VENV_DIR:-jax-models.venv}"
-PYTHON="${PYTHON:-"$(which python)"}"
```

Comment:
Is there a special reason we chose `python` instead of `python3` as default?

Reply:
Yes, there are different versions of Python3 that work with certain frameworks. python3.10 works with TF, JAX but PyTorch needs python3.11 for torch-mlir to work.

Reply:
I would prefer to keep this.

Reply:
Reverted